.gitignore

```diff
@@ -17,15 +17,5 @@
 vendor/
 cmd/device-gpio
 
-# snap files
-*.snap
-*.assert
-prime/
-stage/
-parts/
-squashfs-root/
-coverage.out
-VERSION
```

Comment:
The last two aren't snap related.

Reply:
These were listed as related to snap all over the place:
- https://github.com/edgexfoundry/device-rfid-llrp-go/pull/261/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947
- https://github.com/edgexfoundry/app-rfid-llrp-inventory/pull/238/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947

Reply:
The lists inside .gitignore files are poorly structured.

The `VERSION` file is used by the Makefile regardless of the packaging.
The `coverage.out` file is the report from `go tool cover`